src/theme.js

```diff
@@ -277,6 +277,13 @@ const theme = {
     l: "1024px",
     xl: "1440px", // Used as the max-width
   },
+  footerBreakpoints: {
```

Comment:
Agree `footerBreakpoints` will likely be specific to the Footer component. Let's move these out of theme & keep in that component for now.